Proposed edits to requesting access page (#451)
ASFHyP3/hyp3-docs: docs/using/requesting_access.md

!!! tip "Existing HyP3 users do not need to request access" 

    Existing On Demand users (i.e. users who signed up before June 17, 2024) do *not* need to undergo this process.
    If you are no longer able to process On Demand jobs and you believe you should still have access, you can either 
    submit an access request or email [uso@asf.alaska.edu](mailto:uso@asf.alaska.edu "uso@asf.alaska.edu") for more information.

## Workshops and Tutorials

If you are leading a workshop or tutorial and your participants will require access to HyP3, 
we can provide you with a unique access code to streamline the access request process. Your participants can enter 

> this code in the access request form to receive automatic approval of their account without having to wait for the 
> usual approval process. 

Reviewer: What about something like this to reduce verbosity?
```suggestion
this code in the access request form to receive immediate, automatic approval.
```
